content/en/agent/docker/integrations.md

```diff
@@ -70,6 +71,8 @@ labels:
   com.datadoghq.ad.instances: '[<INSTANCE_CONFIG>]'
 ```
 
+**Note**: As a best practice, Datadog recommends using unified service tagging in containerized environments. Unified service tagging ties Datadog telemetry together through the use of three standard tags: `env`, `service`, and `version`. To learn how to configure your environment with unified tagging, refer to the dedicated [unified service tagging][11] documentation.
```

Comment:
this note applies to all different configurations (e.g. Dockerfile, docker-compose, etc), but it's currently placed between them. Suggesting to put at the end before the "Examples" section begins

Reply:
Good catch, I'll move at the end before examples.